geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/AccumuloFeatureStore.scala

```diff
@@ -50,6 +52,8 @@ class AccumuloFeatureStore(val dataStore: AccumuloDataStore, val featureName: Na
         val feature = iter.next()
         val newFeature = fw.next()
 
+        minMaxVisitorO.foreach ( _.visit(feature) )
```

Comment:
Hmm, the codes here are wrong (different from my local branch) for some reason. It should be:
minMaxVisitorO.foreach ( _.visit(feature) )
I'going to change.